CoJezdzi/AppFramework/_Networking/NetworkLogger.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+class NetworkLogger {
+    class func logRequest(_ request: URLRequest, enabled: Bool = true) {
+        guard enabled else {
+            return
+        }
+        
+        let headerString = request.allHTTPHeaderFields?.headerString()
+        let httpBodyString = String(data: request.httpBody ?? Data(), encoding: .utf8) ?? ""
+        
+        print("""
```

Comment:
Could you change that to 'os_log' or 'debugPrint'?

Reply:
sure